### DISCUSSION TITLE
Welcome to Omex-landing-page Discussions!

### DISCUSSION THREAD

**Discussion by ghost** [student-to-dev/Omex-landing-page#17] [Announcements]
## Dėl projekte naudojamų paveiksliukų ir šriftų
  
  Sveiki, @belauzas 
  Projekto paruošimui reikalingi jame naudojami paveiksliukai bei šriftai. Gal galėtumėte juos gauti ir atsiųsti?

**Comment by belauzas** [comment]
Ok, paklausiu kliento. Pranešiu kai atsakys 👍

**Reply by eyyMinda** [reply]
Nu jau visai toks klientas, nei ten jam to puslapio reikia nieko... Apsileidęs ajaj.

**Comment by belauzas** [comment]
https://github.com/student-to-dev/Omex-landing-page/tree/develop/CLIENT/images
